deploy/scripts/Create-IomtWebJobsEnvironment.ps1

```diff
@@ -0,0 +1,109 @@
+<#
+.SYNOPSIS
+Creates a new IoMT FHIR Connector for Azure without using Stream Analytics
+.DESCRIPTION
+#>
+param
```

Comment:
It might actually be a nice feature to be able to specify the resource group name instead of assuming the resource group will be named the same as all of the resources.

Reply:
that's a good idea. updated